composer.json

```diff
@@ -10,25 +10,25 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
         "ext-rdkafka": "^5.0",
-        "arquivei/events-sender": "^3.2",
-        "arquivei/laravel-health-checker": "^2.2",
-        "arquivei/laravel-prometheus-exporter": "^2.0",
-        "arquivei/php-kafka-consumer": "^2.3",
-        "laravel/framework": "^8.50",
-        "monolog/monolog": "^2.3",
-        "spatie/data-transfer-object": "^3.5"
+        "arquivei/events-sender": "3.2.1",
+        "arquivei/laravel-health-checker": "2.2.0",
+        "arquivei/laravel-prometheus-exporter": "2.0.0",
+        "arquivei/php-kafka-consumer": "2.3.1",
+        "laravel/framework": "8.77.1",
+        "arquivei/log-adapter": "1.0.0",
+        "spatie/data-transfer-object": "3.7.2"
     },
     "require-dev": {
-        "barryvdh/laravel-ide-helper": "^2.9",
-        "fakerphp/faker": "^1.13",
-        "mockery/mockery": "^1.4",
-        "nunomaduro/collision": "^5.6",
-        "phpunit/phpunit": "^9.5",
-        "squizlabs/php_codesniffer": "^3.6",
-        "phpstan/phpstan": "^0.12",
-        "phpstan/phpstan-phpunit": "^0.12.8"
+        "barryvdh/laravel-ide-helper": "2.10.0",
+        "fakerphp/faker": "1.17.0",
+        "mockery/mockery": "1.4.4",
+        "nunomaduro/collision": "5.10.0",
+        "phpunit/phpunit": "9.5.11",
+        "squizlabs/php_codesniffer": "3.6.2",
+        "phpstan/phpstan": "0.12.99",
+        "phpstan/phpstan-phpunit": "0.12.22"
```

Comment:
I believe there is a newer version than the used here.
Check the [laravel-skeleton-api](https://github.com/arquivei/laravel-skeleton-api) for reference or just check in [packagist](https://packagist.org/).

Reply:
sim, atualizado.